code/modules/examine/descriptions/telecomms.dm

```diff
@@ -30,7 +30,7 @@
 	Exonets at the root node(s), and is typically arranged in a tree structure.  The root node(s) are generally government-owned and are very secure \
 	and resilient to failure.<br>\
 	<br>\
-	This node is privately owned and maintained by Nanotrasen, and allows the colonists of the Northern Star to have access to the Exonet."
+	This node is privately owned and maintained by Nanotrasen, and allows the users on Virgo Station to have access to the Exonet."
```

Comment:
Should say "allows inhabitants of the star system to have access to the exonet." The telecomms services more than just our station. It affects the planet Virgo-Prime, Central Command, _and_ the station we work on.